test/queue.jl

```diff
@@ -25,6 +25,7 @@ end
         sqs_send_message(queue, "Hello!", options...)
         message = sqs_receive_message(queue)
 
+        @test haskey(message, :id) && isempty(message[:id])
```

Comment:
I think you meant to put:

```suggestion
        @test haskey(message, :id) && !isempty(message[:id])
```

Reply:
hahaha yes that's right. Nice catch

Reply:
Fixed